examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -137,12 +139,15 @@ def preprocess(self, requests):
                 context = question_context["context"]
                 inputs = self.tokenizer.encode_plus(question, context, max_length=int(max_length), pad_to_max_length=True, add_special_tokens=True, return_tensors="pt")
             input_ids = inputs["input_ids"].to(self.device)
+            attention_mask = inputs["attention_mask"].to(self.device)
             if input_ids.shape is not None:
-                if input_batch is None:
-                    input_batch = input_ids
+                if input_ids_batch is None:
+                    input_ids_batch = input_ids
+                    attention_mask_batch = attention_mask
                 else:
-                    input_batch = torch.cat((input_batch, input_ids), 0)
-        return input_batch
+                    input_ids_batch = torch.cat((input_ids_batch, input_ids), 0)
+                    attention_mask_batch = torch.cat((attention_mask_batch, attention_mask), 0)
```

Comment:
input_ids_batch is always None. it seems that "else" condition will never be hit.

Reply:
@lxning it is aggregating a batch of input_ids and attention_masks, it get used after the first time/request where input_batch_ids is not None anymore.